packages/app/package.json

```diff
@@ -62,5 +64,12 @@
       "last 1 safari version"
     ]
   },
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "proxy": {
```

Comment:
So we're probably gonna want to remove this in the future in favor of a more integrated frontend/backend dev environment, but should be fine for now.